tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -276,27 +291,284 @@ test.describe( 'Set up accounts', () => {
+				} );
+			} );
+		} );
+
+		test.describe( 'Connect Merchant Center account', () => {
+			test.beforeAll( async () => {
+				await Promise.all( [
+					setUpAccountsPage.mockAdsAccountsResponse( ADS_ACCOUNTS ),
+
+					// Mock merchant center as not connected.
+					setUpAccountsPage.mockMCNotConnected(),
+
+					// Mock merchant center has accounts
+					setUpAccountsPage.mockMCHasAccounts(),
+				] );
+
+				await setUpAccountsPage.goto();
+			} );
+
+			test.describe( 'connect to an existing account', () => {
+				test( 'should see "Select an existing account" title', async () => {
+					const selectAccountTitle =
+						setUpAccountsPage.getSelectExistingMCAccountTitle();
+					await expect( selectAccountTitle ).toContainText(
+						'Connect to existing Merchant Center account'
+					);
+				} );
+
+				test( 'should see "Or, create a new Merchant Center account" text', async () => {
+					const mcFooter = setUpAccountsPage.getMCCardFooter();
+					await expect( mcFooter ).toContainText(
+						'Or, create a new Merchant Center account'
+					);
+				} );
+
+				test( 'should see "Connect" button', async () => {
+					const connectButton = setUpAccountsPage.getConnectButton();
+					await expect( connectButton ).toBeEnabled();
+				} );
+
+				test( 'should see "Continue" button is disabled', async () => {
+					const continueButton =
+						setUpAccountsPage.getContinueButton();
+					await expect( continueButton ).toBeDisabled();
+				} );
+
+				test( 'select MC Account 2 and click "Connect" button should see Merchant Center "Connected"', async () => {
+					await Promise.all( [
+						// Mock Merchant Center create accounts
+						setUpAccountsPage.mockMCCreateAccountWebsiteNotClaimed(),
+
+						// Mock Merchant Center as connected with ID 12345
+						setUpAccountsPage.mockMCConnected( 23456 ),
+					] );
+
+					// Select MC Account 2 from the options
+					const mcAccountsSelect =
+						setUpAccountsPage.getMCAccountsSelect();
+					await mcAccountsSelect.selectOption( {
+						label: 'MC Account 2 ・ https://example.com (23456)',
+					} );
+
+					// Click connect button
+					const connectButton = setUpAccountsPage.getConnectButton();
+					await connectButton.click();
+
+					const mcConnectedLabel =
+						setUpAccountsPage.getGoogleComboConnectedLabel();
+					await expect( mcConnectedLabel ).toContainText(
+						'Connected'
+					);
+
+					const mcDescriptionRow =
+						setUpAccountsPage.getMCDescriptionRow();
+					await expect( mcDescriptionRow ).toContainText(
+						'Merchant Center ID: 23456'
+					);
+				} );
+			} );
+
+			test.describe( 'click "Or, create a new Merchant Center account"', () => {
+				test.beforeAll( async () => {
+					await Promise.all( [
+						// Mock merchant center as not connected.
+						setUpAccountsPage.mockMCNotConnected(),
+
+						// Mock merchant center has accounts
+						setUpAccountsPage.mockMCHasAccounts(),
+					] );
+
+					await setUpAccountsPage.goto();
+				} );
+			} );
```

Comment:
This test group should be removed as it doesn't have any tests.

Reply:
👍